src/test/java/org/openstreetmap/atlas/geography/atlas/pbf/OsmPbfNodeToAtlasItemTestRule.java

```diff
@@ -0,0 +1,77 @@
+package org.openstreetmap.atlas.geography.atlas.pbf;
+
+import org.openstreetmap.atlas.geography.atlas.Atlas;
+import org.openstreetmap.atlas.utilities.testing.CoreTestRule;
+import org.openstreetmap.atlas.utilities.testing.TestAtlas;
+
+/**
+ * {@link OsmPbfNodeToAtlasItemTest} test cases.
+ *
+ * @author mgostintsev
+ */
+public class OsmPbfNodeToAtlasItemTestRule extends CoreTestRule
+{
+    @TestAtlas(loadFromJosmOsmResource = "noRelationNoTagsNoIntersectionAtlas.osm")
```

Comment:
As the `loadFromJosmOsmResource` uses the PBF loading to generate the test Atlas every time, it cannot be used to test the PBF loading... I would recommend saving all those Atlas files to textAtlas one time, and then loading them using `loadFromTextAtlas`. Like this the pbf loading is out of the equation, and you keep the same test set!

Reply:
Fair point, let me update it!

Reply:
Not sure I fully agree here. If we use text atlases, then we're just loading an Atlas using PackedAtlasBuilder under the covers. This doesn't test PBF entity to Atlas Item conversion in any way. I think it actually makes sense to use .osm files here to test that code flow. Thoughts?

Reply:
Oh I did not realize that the test depended on having something loaded using the PBF loader. In that case this is fine.
Mind adding a comment explaining why `loadFromJosmOsmResource` is actually right here?

Reply:
Sure, let me add one.